.golangci.yml

```diff
@@ -66,10 +69,6 @@ issues:
   - path: test/pipelinerun_test\.go
```

Comment:
not sure why is this here? its much smaller in size compared to `pkg/reconciler/pipelinerun/pipelinerun_test.go` with less 1,100 lines.

Reply:
I'm not sure what's happening with this one either -- when I remove the exclusion, the `unused` linter falsely identifies several in-use functions as unused. 🤷🏾